.github/workflows/release_signature_aggregator.yml

```diff
@@ -0,0 +1,75 @@
+# Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+name: Release signature-aggregator
+
+on:
+  push:
+    tags:
+      - "signature-aggregator/*"
+
+jobs:
+  release:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      # The GO_VERSION must be set explicitly to be used in the Dockerfile.
+      - name: Set Go version
+        run: |
+          source ./scripts/versions.sh
+          echo GO_VERSION=$GO_VERSION >> $GITHUB_ENV
```

Comment:
I don't think this bit is necessary anymore now that the go.mod file can contain the patch version and `actions/setup-go` below accepts it

Reply:
great catch! thank you! addressed in https://github.com/ava-labs/awm-relayer/pull/435/commits/7c289ce0d9732a239b0cc883c29e9943c02d15d1, and validated in https://github.com/ava-labs/awm-relayer/actions/runs/10724002269 and https://github.com/ava-labs/awm-relayer/actions/runs/10724006401